launchdarkly-android-client-sdk/src/main/java/com/launchdarkly/sdk/android/ConnectivityManager.java

```diff
@@ -76,8 +76,6 @@ class ConnectivityManager {
     // source we're using, like "if there was an error, update the ConnectionInformation."
     private class DataSourceUpdateSinkImpl implements DataSourceUpdateSink {
         private final ContextDataManager contextDataManager;
-        private final AtomicReference<ConnectionMode> connectionMode = new AtomicReference<>(null);
-        private final AtomicReference<LDFailure> lastFailure = new AtomicReference<>(null);
```

Comment:
For reviewers: These pieces of data duplicated what was held in the ConnectionInformationState member of the ConnectivityManager, so removed duplication to reduce chance of being out of sync.